lib/tasks/normalise_whitespace.rake

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+desc "Run \`normalise_whitespace\` on all relevant strings in the database"
```

Comment:
```suggestion
desc "Normalise the white space in a number of fields for patients and parents to ensure consistency."
```

Not a big deal since it's likely we will delete this task after it's run, but the current description doesn't explain what the task does (and relies on the user to know what the `normalise_whitespace` method does).